functions/Get-DbaHelpIndex.ps1

```diff
@@ -115,7 +119,7 @@ function Get-DbaHelpIndex {
 		[string]$ObjectName,
 		[switch]$IncludeStats,
 		[switch]$IncludeDataTypes,
-		[switch]$FormatResults,
+		[switch]$Raw,
```

Comment:
This should have `FormatResults` as an Alias to `Raw` and will prevent commands from failing.

Reply:
Well actually no, this would be the opposite of what Raw does so...never mind.